hms-app/src/actions/SkillActions.tsx

```diff
@@ -1,11 +1,16 @@
+import { IPublicClientApplication } from '@azure/msal-browser'
+import { getIdToken } from '../common/actionAuth'
+
 const core_url = process.env.REACT_APP_CORE_URL
 
-export const getListOfSkills = () => {
+export const getListOfSkills = async (instance: IPublicClientApplication) => {
+  const idToken = await getIdToken(instance)
   return fetch(`${core_url}/skills`, {
     method: 'GET',
     headers: {
       Accept: 'application/json',
       'Content-Type': 'application/json',
+      Authorization: `Bearer ${idToken}`,
     },
```

Comment:
this chunk is duplicated all over the place, we could extract it to a single const function

Reply:
it could include the `method` bit as well... maybe something like 

```
const buildFetchOptions = (method: String, token: String) => ({
    method: method,
    headers: {
      Accept: 'application/json',
      'Content-Type': 'application/json',
      Authorization: `Bearer ${token}`,
    }
});
```